### ISSUE TITLE
[Feature Request] Project Dependencies

### ISSUE THREAD

**Issue by Alex7Li** [kochrt/markwhen#87]
Hi, thanks so much for this very nice tool. One thing that is missing is the ability to mark one task as dependent on another task. I think this is a fairly common feature and it can be quite useful as well. For example, I am imagining that you could have something like

```
11/2022-12/2022: Create Project Plan #Plan
2/2023-12/2023: Do project #after:Plan
```

and then you will get an arrow from the end of the "Create Project Plan" bar to the start of the "Do project" bar. Though I'm not sure how hard it would be to implement, I think it would be quite useful.

**Comment by emeyedeejay** [comment]
Another request for this ... would be very helpful!

**Comment by deverman** [comment]
Yeah I was reading the documentation and didn’t see any mention of dependencies think it would be useful.

**Comment by frazenshtein** [comment]
This would be a super feature, as it is sorely lacking when several different projects depend on one other.

**Comment by kochrt** [comment]
Is the arrow the most important part? I'm trying to understand how this is different from [relative dates with event ids](https://docs.markwhen.com/syntax/dates-and-ranges.html#event-ids)

**Comment by frazenshtein** [comment]
> Is the arrow the most important part? I'm trying to understand how this is different from [relative dates with event ids](https://docs.markwhen.com/syntax/dates-and-ranges.html#event-ids)

Event ids is a great thing, but it's about something else - it's the ability to correctly position single events relative to each other. For events that are in the same project or next to each other, this works well. But when you need to indicate a dependency between different projects that include many other nested projects or events, it becomes impossible to read and see it visually. 

Therefore, it seems to me that the arrow here plays a key role - to remove the need to read the file and see the dependency in a visual representation. This significantly reduces the cognitive load and allows you to naturally see dependencies.

It also turned out that event IDs are displayed incorrectly in the VScode plugin. But this probably needs to be reported separately.

**Comment by kochrt** [comment]
https://docs.markwhen.com/syntax/dates-and-ranges.html#dependencies

Dependent lines have been added, with a setting in the bottom left corner to turn them off
